.github/workflows/buildtest.yml

```
@@ -181,9 +188,33 @@ jobs:
           cd $GITHUB_WORKSPACE/build/tests/install_test
           sh test_install.sh
 
+  clang18_verify_schema:
+    needs: clang18_build
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/download-artifact@v4
+        with:
+          name: clang18_build-schema
+          path: ${{ github.workspace }}
+      - name: json-yaml-validate
+        uses: GrantBirki/json-yaml-validate@v3.2.1
+        with:
+          base_dir: .
```

```suggestion
          base_dir: "tests/input_files/"
```

?

I guess the special extension below will also work.